### ISSUE TITLE
Powers for unit 'in' are incorrect

### ISSUE THREAD

**Issue by matthewbryant95** [josdejong/mathjs#1455]
Evaluation of `1/2 in` results in `0.5 in^-1`. For any other unit, `1/2 _unit_` evaluates to `0.5 _unit_`.

**Comment by gwhitney** [comment]
Wow. This is still true, and definitely a bug. Presumably some side effect of the fact that `10 cm in feet` produces 0.328 feet, but nevertheless, this should either report an error or produce a sensible result, not calmly produce nonsense.

**Comment by nkumawat34** [comment]
can i work on this issue

**Comment by nkumawat34** [comment]
can you explain this issue little bit more with example.

**Comment by josdejong** [comment]
Thanks @nkumawat34 

```js
console.log(math.evaluate('1/2 kg').toString()) 
// actual: 0.5 kg

console.log(math.evaluate('1/2 in').toString()) 
// actual:   0.5 in^-1 
// expected: 0.5 in
```

So, `1/2 in` is evaluated as `1 / (2 * in)` instead of `(1/2) * in`. This may be hard to pick up as a first issue, it requires diving into the expression parser and it's special rules around implicit multiplication and special parsing of "in" which is both a unit (inch) and an operator (like `2 meter in inch`).

**Comment by nkumawat34** [comment]
Please check my PR https://github.com/josdejong/mathjs/pull/3315

**Comment by josdejong** [comment]
Neerja's fix is now published now in `v13.2.2`.
